sdk/keyvault/Azure.Security.KeyVault.Keys/src/JsonWebKey.cs

```diff
@@ -120,6 +120,250 @@ public JsonWebKey(string keyId, string keyType, IList<string> keyOperations)
         /// HSM Token, used with "Bring Your Own Key"
         /// </summary>
         public byte[] T { get; set; }
+
+        internal override void ReadProperties(JsonElement json)
+        {
+            if (json.TryGetProperty("kid", out JsonElement kid))
+            {
+                KeyId = kid.GetString();
+            }
+            if (json.TryGetProperty("kty", out JsonElement kty))
+            {
+                KeyType = KeyTypeExtensions.ParseFromString(kty.GetString());
+            }
+            if (json.TryGetProperty("key_ops", out JsonElement key_ops))
+            {
+                foreach (var element in key_ops.EnumerateObject())
```

Comment:
Is it ok for this to throw if key_ops is not a property name of a JSON object or does it need to be guarded against that?

Reply:
Issue https://github.com/Azure/azure-sdk-for-net/issues/6474 to follow up with service team